README.md

```diff
@@ -8,22 +8,30 @@ In order to run this demo, you will need a Dolby.io account. If you don't alread
 
 ## Features and roadmap
 
-- [x] Publish and view streams
-- [x] Invite viewers to watch your stream
-- [x] Mute audio and video both as publisher and viewer
-- [x] Screen sharing
+### RTS Publisher
+
+- [x] Publish streams
 - [x] Publish with simulcast (only on Chrome)
 - [x] Microphone and camera device selection
+- [x] Invite viewers to watch your stream
+- [x] Screen sharing
+- [x] Mute audio and video
 - [x] Resolution, bandwidth, codec, and bitrate selection
-- [x] Observing stream statistics
-- [ ] Ability to stream from two or more sources
+- [x] Ability to stream from two or more sources
+- [x] Observing stream statistics for each simulcast layer
 - [ ] Ability to have multiple publishers
-- [ ] Grid layout for video sources
 - [ ] Recording your streams
+
+### RTS Viewer
+
+- [x] View streams
+- [x] Mute audio and video
+- [x] Observing stream statistics
+- [x] Select incoming stream quality with simulcast
+- [x] Grid layout for video sources
```

Comment:
We don't have a grid layout anymore, in V2 we have list layout with main stream projected in large view.